scripts/check_format.sh

```diff
@@ -34,7 +34,7 @@ while read -d '' filename; do
       FAILED=1
     fi
   fi
-done < <(find ${ROOTS} $(emit_prunes) -name '*.cpp' -print0 -o -name '*.h' -print0)
+done < <(find ${ROOTS} $(emit_prunes) -name '*.cpp' -print0 -o -name '*.h' -print0 -o -name '*.cu' -print0)
```

Comment:
Sorry, my bad. I should have mentioned that we also have *.cuh files in our repo. Can you please add that too?